quantecon_book_networks/data.py

```diff
@@ -108,9 +108,9 @@ def introduction():
     data_file = pkg_resources.resource_stream(__name__, data_file)
     crude_oil = pd.read_csv(data_file, dtype={'product_id': str})
 
-    exporters = crude_oil.groupby(by=["location_code"]).sum().sort_values("export_value", ascending=False)[:10].index
+    exporters = crude_oil.groupby(by=["location_code"]).sum(numeric_only=True).sort_values("export_value", ascending=False)[:10].index
```

Comment:
@HumphreyYang from a `data person` perspective this makes me a little nervous we should perhaps select the columns that are `value` columns via selection in the `crude_oil` data frame such as `crude_oil['export_value'].groupby(by ....`

Reply:
Hi @mmcky, 

I have committed to the requested change. Please kindly merge if it looks good to you. I will double-check if the future warning is resolved in the intro lecture once this is merged. 

Many thanks in advance.